src/locales/ko/mystery-encounters/the-strong-stuff-dialogue.json

```
@@ -1,21 +1,21 @@
 {
-  "intro": "It's a massive {{shuckleName}} and what appears\nto be a large stash of... juice?",
-  "title": "The Strong Stuff",
-  "description": "The {{shuckleName}} that blocks your path looks incredibly strong. Meanwhile, the juice next to it is emanating power of some kind.\n\nThe {{shuckleName}} extends its feelers in your direction. It seems like it wants to do something...",
-  "query": "What will you do?",
+  "intro": "커다란 {{shuckleName}}를 만났다!\n마시고 있는 심상치 않아보이는 주스는 뭘까…?",
+  "title": "강력한 물질",
+  "description": "당신을 막아선 {{shuckleName}}[[는]] 엄청나게 강해보입니다. 게다가, 그 옆에 있는 주스는 무언가 기운이 느껴지는 것 같습니다.\n\n{{shuckleName}}가 당신 쪽으로 발을 내밉니다. 무언가 하려는 모양입니다…",
+  "query": "어떻게 하시겠습니까?",
   "option": {
     "1": {
-      "label": "Approach the {{shuckleName}}",
-      "tooltip": "(?) Something awful or amazing might happen",
-      "selected": "You black out.",
-      "selected_2": "@f{150}When you awaken, the {{shuckleName}} is gone\nand juice stash completely drained.${{highBstPokemon1}} and {{highBstPokemon2}}\nfeel a terrible lethargy come over them!$Their base stats were reduced by {{reductionValue}}!$Your remaining Pokémon feel an incredible vigor, though!\nTheir base stats are increased by {{increaseValue}}!"
+      "label": "{{shuckleName}}에게 다가간다",
+      "tooltip": "(?) 끔찍하거나 놀라운 일",
+      "selected": "당신은 기절해버렸다……….",
+      "selected_2": "@f{150}정신을 차리고 나니, {{shuckleName}}[[는]] 사라지고\n주스도 완전히 사라졌다.${{highBstPokemon1}}[[과]] {{highBstPokemon2}}[[는]]\n굉장히 무기력해졌다!$두 포켓몬의 기본 능력치가 {{reductionValue}}만큼 감소했다!$하지만 남은 다른 포켓몬들은 놀라울만큼 활력이 넘친다!\n다른 포켓몬의 기본 능력치가 {{increaseValue}}만큼 증가했다!"
```

```suggestion
      "selected_2": "@f{150}정신을 차리고 나니, {{shuckleName}}[[는]] 사라지고\n주스도 완전히 비었다.${{highBstPokemon1}}[[과]] {{highBstPokemon2}}[[는]]\n굉장히 무기력해졌다!$두 포켓몬의 기본 능력치가 {{reductionValue}}만큼 감소했다!$하지만 다른 포켓몬들은 놀라울만큼 활력이 넘친다!\n다른 포켓몬의 기본 능력치가 {{increaseValue}}만큼 증가했다!"
```
1) drain의 의미를 조금 살려 보았습니다.
2) remaining의 사전적 의미는 '남은'이지만 한국어 문장상 '다른'만 써도 의미가 통할 듯합니다.